VERSION

```diff
@@ -1 +1 @@
-2.16.4.dev0
+2.17.0.dev0
```

Comment:
This file should not be changed manually.
```suggestion
2.16.4.dev0
```

Reply:
Discussed offline.

Reply:
Agreed. Following the pattern used last year: https://github.com/aws/sagemaker-python-sdk/pull/1143/commits/dcd5d437bf2338ddba5c49cf154fda83482a1264

1.45.0 was still published for PyPI: https://pypi.org/project/sagemaker/1.45.0/